Add configurable max connections limit for the omni-executor RPC server

silva-fj opened the pull request:
This allows better control over server resource usage and scaling capabilities.

silva-fj committed:
feat(rpc-server): add max connections configuration

Add configurable max connections limit for the RPC server via the
RPC_SERVER_MAX_CONNECTIONS environment variable with a default value
of 100 connections. This allows better control over server resource
usage and scaling capabilities.

kziemianek commented on tee-worker/omni-executor/rpc-server/src/server.rs:
```diff
@@ -73,7 +73,12 @@ pub async fn start_server<
 	let address = format!("0.0.0.0:{}", port);
-	let server = Server::builder().build(address.parse::<SocketAddr>()?).await?;
+	let max_connections: u32 =
+		env::var("RPC_SERVER_MAX_CONNECTIONS").unwrap_or("100".to_string()).parse()?;
```
Should we prefix all env variables with `OE` or `OMNI_EXECUTOR` ?

silva-fj replied:
It sounds good

silva-fj committed:
fix: prefix environment variables with OE_

Updates environment variable names in executor-worker and rpc-server
to use a consistent OE_ prefix, helping to prevent namespace
collisions with other applications.

silva-fj committed:
Merge branch 'dev' into add-max-connections-rpc-server-config
